app/cdap/components/WrangleHome/Components/OngoingDataExploration/index.tsx

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import React, { useEffect, useState } from 'react';
+import { Box } from '@material-ui/core/';
+import { generateDataForExplorationCard } from './utils';
+import MyDataPrepApi from 'api/dataprep';
+import { Link } from 'react-router-dom';
+import { getCurrentNamespace } from 'services/NamespaceStore';
+import OngoingDataExplorationCard from '../OngoingDataExplorationCard';
+import { switchMap } from 'rxjs/operators';
+import { forkJoin } from 'rxjs/observable/forkJoin';
+
+const OngoingDataExploration = (props) => {
+  const [ongoingExpDatas, setOngoingExpDatas] = useState<any>([]);
+  const [finalArray, setFinalArray] = useState([]);
+
+  const getOngoingData = () => {
+    MyDataPrepApi.getWorkspaceList({
```

Comment:
please comment this to describe whats going on

Reply:
Done